plugins/woocommerce-admin/client/homescreen/activity-panel/orders/test/index.js

```diff
@@ -20,6 +22,17 @@ jest.mock( '@wordpress/data', () => {
 	};
 } );
 
+jest.mock( '@woocommerce/tracks', () => {
+	// Require the original module to not be mocked...
+	const originalModule = jest.requireActual( '@woocommerce/tracks' );
+
+	return {
+		__esModule: true, // Use it when dealing with esModules
+		...originalModule,
+		recordEvent: jest.fn(),
+	};
+} );
```

Comment:
It would be nice to put this in `@woocommerce/internal-js-tests` so we didn't have to repeat this everywhere.

I don't think we'd ever want to actually make the actual `recordEvent` call in the unit tests, so you could probably just always mock it by default.